man/io_uring_prep_timeout.3

```diff
@@ -77,7 +77,7 @@ The specified timeout occurred and triggered the completion event.
 .TP
 .B -EINVAL
 One of the fields set in the SQE was invalid. For example, two clocksources
-where given, or the specified timeout seconds or nanoseconds where < 0.
+were given, or the specified timeout seconds or nanoseconds were < 0.
```

Comment:
This and the next hunk should arguably be a separate commit, it has nothing to do with the current man page being added.